static/data/featureContentData.json

```diff
@@ -0,0 +1,26 @@
+[
+  {
+    "title": "Introduction to SCS",
+    "body": "Get to know SCS better and learn about the background.",
+    "url": "/docs",
+    "buttonText": "Get Started"
+  },
+  {
+    "title": "Releases",
+    "body": "SCS is currently in Release 5. Check out the latest Release Notes.",
+    "url": "/docs/releases/Release5",
+    "buttonText": "Learn More"
+  },
+  {
+    "title": "Frequently Asked Questions",
+    "body": "You are curious what SCS is all about, what it can do and what it cant?",
+    "url": "/docs/faq",
+    "buttonText": "Get Answers"
+  },
+  {
+    "title": "Existing Public Clouds",
+    "body": "There are public SCS compliant clouds in production.",
```

Comment:
```suggestion
    "body": "There are SCS compliant public clouds in production.",
```
ChatGPT:
> 
> Both "public SCS compliant clouds" and "SCS compliant public clouds" are grammatically correct, but they slightly differ in emphasis:
> 
> Public SCS Compliant Clouds: This phrasing emphasizes that the clouds are public first, with SCS compliance as a secondary characteristic. It suggests that within the category of public clouds, you are specifically referring to those that are SCS compliant.
> 
> SCS Compliant Public Clouds: This phrasing puts the emphasis on SCS compliance, with the public nature of the clouds as a secondary characteristic. It implies that among SCS compliant clouds, you are specifically talking about the ones that are public.
> 
> The choice between the two would depend on what aspect you want to highlight. If the primary focus is on the fact that these are public clouds, which also happen to be SCS compliant, the first phrasing is more appropriate. If the emphasis is more on SCS compliance in the context of public clouds, then the second phrasing is more suitable.

This is probably not worth the change but maybe should be different ordering :laughing: